homeassistant/components/pi_hole/sensor.py

```diff
@@ -58,6 +58,7 @@ def __init__(
 
         self._attr_name = f"{name} {description.name}"
         self._attr_unique_id = f"{self._server_unique_id}/{description.name}"
+        self._attr_icon = description.icon  # Necessary to overwrite inherited value
```

Comment:
```suggestion
        if description.icon:
                # Necessary to overwrite inherited value
                self._attr_icon = description.icon
```
We only want to overwrite this if an icon is defined in the description?

Reply:
Can be removed, now the description provides the default/customized version.

```suggestion
```